# Patch to make the program properly compile against gcc 4.1
# See http://bugs.debian.org/357404

Index: rrootage-0.23a/src/foecommand.h
===================================================================
--- rrootage-0.23a.orig/src/foecommand.h	2007-08-29 19:48:31.000000000 +0000
+++ rrootage-0.23a/src/foecommand.h	2007-08-29 19:50:14.000000000 +0000
@@ -39,10 +39,10 @@
   
   virtual void doChangeDirection(double d);
   virtual void doChangeSpeed(double s);
-  virtual void FoeCommand::doAccelX(double ax);
-  virtual void FoeCommand::doAccelY(double ay);
-  virtual double FoeCommand::getBulletSpeedX();
-  virtual double FoeCommand::getBulletSpeedY();
+  virtual void doAccelX(double ax);
+  virtual void doAccelY(double ay);
+  virtual double getBulletSpeedX();
+  virtual double getBulletSpeedY();
   
  private:
   struct foe *foe;
